src/components/CardSelection/CardSelection.tsx

```diff
@@ -0,0 +1,160 @@
+import { Checkbox as AntCheckbox, Radio as AntRadio, RadioChangeEvent } from 'antd'
+import { ComponentType, ReactElement, ReactNode, useCallback, useMemo } from 'react'
+import classnames from 'classnames'
+
+import { InputType, Layout } from './types.ts'
+import { CardSelectionProps } from './props.ts'
+import { Checkbox } from '../Checkbox'
+import { Icon } from '../Icon'
+import { IconComponent } from '../Icon/Icon.props.ts'
+import { Radio } from '../RadioGroup/components/Radio'
+import styles from './CardSelection.module.css'
+
+const DIV = 'div' as unknown as ComponentType
+
+export const defaults = {
+  gap: 24,
+  layout: Layout.Vertical,
```

Comment:
Are you sure that the default case is Vertical and not Horizontal? We will use this component horizontally in most cases